cl/lib/test_helpers.py

```diff
@@ -109,7 +109,7 @@ def setUp(self) -> None:
 class SitemapTest(TestCase):
 
     sitemap_url: str
-    item_qs: QuerySet
```

Comment:
Since this it's not needed anymore, might be good to remove it from imports on line 5:
`from django.db.models import QuerySet`

Reply:
Thanks, got it.